index.js

```diff
@@ -26,5 +26,6 @@ module.exports = {
     combineMeshes : require('./lib/combineMeshes'),
     combinePrimitives : require('./lib/combinePrimitives'),
     removeUnusedVertices : require('./lib/removeUnusedVertices'),
-    OptimizationStatistics : require('./lib/OptimizationStatistics')
+    OptimizationStatistics : require('./lib/OptimizationStatistics'),
+    gltfPipeline : require('./lib/gltfPipeline')
```

Comment:
I'm starting to wonder if `optimizeGltf` is a better name here now that it doesn't input and output files.